employee-rostering/src/main/java/employeerostering/employeerostering/DayOffRequest.java

```diff
@@ -0,0 +1,36 @@
+package employeerostering.employeerostering;
```

Comment:
Good idea to do it like the other wb examples. So no license file I guess.

I do have licenses in the optaplanner examples, but there it's less of an issue because the IDE automatically collapses them?